.github/docker/check_result.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+echo "This script should be run in repository root directory."
+
+# If new test cases are added in ClusterTest/run_cluster_test.sh
+# TOTAL_TEST_CASE_COUNT may be updated
+export TOTAL_TEST_CASE_COUNT=2
+export TOTAL_TIMEOUT_IN_SECONDS=3600
+
+echo "GITHUB_RUN_ID=$GITHUB_RUN_ID"
+echo "GITHUB_SHA=$GITHUB_SHA"
+
+echo "Start to downlod dependent libraries"
+curl -s -o scala-library-2.11.12.jar https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.11.12/scala-library-2.11.12.jar
+curl -s -o slf4j-log4j12-1.7.30.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-log4j12/1.7.30/slf4j-log4j12-1.7.30.jar
+curl -s -o slf4j-api-1.7.30.jar https://repo1.maven.org/maven2/org/slf4j/slf4j-api/1.7.30/slf4j-api-1.7.30.jar
+curl -s -o snowflake-jdbc-download.jar https://repo1.maven.org/maven2/net/snowflake/snowflake-jdbc/3.12.2/snowflake-jdbc-3.12.2.jar
```

Comment:
jdbc version can be a variable, and read from sbt file

Reply:
> jdbc version can be a variable, and read from sbt file

This JDBC is used to connect to Snowflake to query the result table. It is no necessary to update frequently.